src/common/vocabulary/actions.js

```diff
@@ -20,6 +21,31 @@ export default function actions(state) {
   };
 
   return {
+    //become a card(becomeACopy takes a playedObject, this takes a card)
```

Comment:
nit (here and below)- Our convention for comments is space before and after the `//` and sentence case.